PP_APP/server/frontend/templates/not_found_404.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+
+<section id="error_404" class="about-section bg-light">
+    <div class="container">
+
+        <h1> Sorry, page requested is not available.</h1>
+        <h1> :( </h1>
+
+    </div>
+</section>
+
+{% endblock content %}
```

Comment:
Great that we have this page. Also something to do after the beta, but we should create some functionality in case mask-rcnn or GV fail for the user to submit their own grading. Or at least let them know what happened.